### PR TITLE
Added sheep dying.

### PR DESCRIPTION
**Pull request by spacechase0**
Needs this first: https://github.com/SpoutDev/Vanilla/pull/72

Signed-off-by: Chase W chasew@silkweaver.com

### CODE REVIEW

File: src/main/java/org/spout/vanilla/material/item/Dye.java [hunk 1]
```diff
@@ -25,11 +25,43 @@
...
+import org.spout.api.inventory.ItemStack;
+import org.spout.api.material.block.BlockFace;
+import org.spout.vanilla.entity.VanillaEntity;
+import org.spout.vanilla.entity.living.passive.Sheep;
+import org.spout.vanilla.entity.living.player.SurvivalPlayer;
 import org.spout.vanilla.material.generic.GenericItem;
 
 public class Dye extends GenericItem {
-
+	final int color;
```
Comment (windy1) [inline src/main/java/org/spout/vanilla/material/item/Dye.java]: Make this private please :)

Comment (spacechase0) [inline src/main/java/org/spout/vanilla/material/item/Dye.java]: Oh, I thought it defaulted to private.

Comment (spacechase0) [inline src/main/java/org/spout/vanilla/material/item/Dye.java]: Done. :)

**Comment by windy1** [comment]
Actually no modifier is something by completely independent. http://stackoverflow.com/questions/215497/in-java-whats-the-difference-between-public-default-protected-and-private

**Pull request merged**